src/test/java/com/wikia/webdriver/testcases/visualeditor/entrypoint/VEDisabledEditorEntryAnonTests.java

```diff
@@ -28,7 +29,7 @@ public void VEDisabledEditorEntryAnonTestsTests_001_CreatePageEntry() {
     String articleName = base.getNameForArticle();
     ArticlePageObject article = new ArticlePageObject().open(articleName);
     VisualEditModePageObject ck = article.createArticleInCKUsingDropdown(articleName);
-    ck.verifyContentLoaded();
+    Assertion.assertTrue(ck.isContentLoaded(), "Content is not loaded");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Content is not loaded" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)